src/hardware/link-mso19/protocol.c

```diff
@@ -56,7 +53,7 @@ SR_PRIV int mso_send_control_message(struct sr_serial_dev_inst *serial,
 
 	w = 0;
 	while (w < s) {
-		ret = serial_write(serial, buf + w, s - w);
+		ret = serial_write_blocking(serial, buf + w, s - w, 10);
```

Comment:
I think this is the type of commit that gsi objects to - the commit line is "remove stale code to get driver to compile ", but here is a change that doesn't really fit with the message.

Reply:
I trying to agree with you, but simply can't. `serial_write` doesn't exist anymore (and thus is stale). So I use the appropriate replacement for the stale API.